arex-storage-web-api/src/main/java/com/arextest/storage/web/controller/AgentRecordingController.java

```diff
@@ -145,7 +141,7 @@ public Response save(@RequestBody AREXMocker requestType) {
     }
     try {
       MDCTracer.addTrace(category, requestType);
-      if (isInvalidRecord(requestType.getRecordId())) {
+      if (cacheService.isInvalidRecord(requestType.getRecordId())) {
```

Comment:
Instead of creating another `CacheService`, you can integrate all this functionality in the class`InvalidIncompleteRecordService`

Reply:
Optimized, call com.arextest.storage.service.InvalidRecordService#isInvalidCase